katharsis-core/src/test/java/io/katharsis/errorhandling/handlers/BaseExceptionMapper.java

```diff
@@ -0,0 +1,20 @@
+package io.katharsis.errorhandling.handlers;
+
+import io.katharsis.errorhandling.ErrorResponse;
+import io.katharsis.errorhandling.mapper.ExceptionMapper;
+
+/**
+ * Created by yuval on 02/03/2017.
+ */
+public abstract class BaseExceptionMapper<E extends Throwable> implements ExceptionMapper<E>{
+
+  @Override
+  public E fromErrorResponse(ErrorResponse errorResponse) {
+    return null;
```

Comment:
What good is a return of `null` ? The idea is to provide some default functionality.